solr/test-framework/src/java/org/apache/solr/SolrTestCaseJ4.java

```diff
@@ -2872,9 +2872,16 @@ public static LBHttpSolrClient getLBHttpSolrClient(String... solrUrls)
   }
 
   /**
-   * This method <i>may</i> randomize unspecified aspects of the resulting SolrClient. Tests that do
-   * not wish to have any randomized behavior should use the {@link
```

Comment:
Hmm I wonder if there should be more randomization here? I think the keyword is `may` but not yet? If we had tests use these methods we could randomize stuff easier?

Reply:
Yeah, I thought it better to fix the docs to match the current code!